src/components/feedback-button.js

```diff
@@ -0,0 +1,39 @@
+import { css } from 'react-emotion'
+import { faCommentDots } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React from 'react'
+import { BLUE, LIGHT_BLUE, WHITE } from '../assets/colors'
+
+const FeedbackLink = styled('a')`
+  align-items: center;
+  background-color: ${LIGHT_BLUE};
+  border-radius: 50%;
+  bottom: 2rem;
+  display: flex;
+  height: 3em;
+  justify-content: center;
+  position: fixed;
+  right: 2rem;
+  width: 3em;
+  &:hover {
+    background-color: ${BLUE};
+  }
+`
```

Comment:
After pulling it out, I'm noticing some other things:

`FeedbackLink` is now a component, but it doesn't set `href` other attributes. When I look at the view, I need to keep it mind that `FeedbackLink` behaves just like an `a` link.

Hmm, it may be more clear then to just pull out the css so we know that's an a link. The benefit of Emotion is that it doesn't expose `feedbackLink` as a class, so it won't conflict w/ any other styles 🎉

Reply:
You'd have the same issue if you just created a separate component called `FeedbackLink` which returned an `a` tag and received an `href` prop. The last company I worked for used to have 6 types of link components in the app and we all knew they took an `href`. React devs kinda get used to these kinds of abstractions. But please do whatever makes you more comfortable. Having a class is totally fine too.